act/utils/data_utils.py

```diff
@@ -1410,3 +1410,112 @@ def calculate_percentages(ds, fields, time=None, time_slice=None, threshold=None
         percentages[i] = j
     ds_percent.close()
     return percentages
+
+
+def convert_2d_to_1d(ds_object, parse=None, variables=None, keep_name_if_one=False,
+                     use_dim_value_in_name=False, dim_labels=None):
+    """
+    Function to convert a single 2D variable into multiple 1D
+    variables using the second dimension in the new variable name.
+
+    ...
```

Comment:
Can we remove the whitespace and 3 . line and leave one space between intro sentence and Parameter header